server/genrest/genrest.go

```diff
@@ -40,6 +40,11 @@ func RegisterHandlers(router *gmux.Router, backend *services.Backend) {
 	router.HandleFunc("/v1beta1/repeat/{info.fString:first/.+}/{info.fChild.fString:second/.+}:pathtrailingresource", rest.HandleRepeatDataPathTrailingResource).Methods("GET")
 	router.HandleFunc("/v1beta1/repeat:bodyput", rest.HandleRepeatDataBodyPut).Methods("PUT")
 	router.HandleFunc("/v1beta1/repeat:bodypatch", rest.HandleRepeatDataBodyPatch).Methods("PATCH")
+	router.HandleFunc("/v1beta1/operations", rest.HandleListOperations).Methods("GET")
+	router.HandleFunc("/v1beta1/{name:operations/.+}", rest.HandleGetOperation).Methods("GET")
+	router.HandleFunc("/v1beta1/{name:operations/.+}:delete", rest.HandleDeleteOperation).Methods("DELETE")
+	router.HandleFunc("/v1beta1/{name:operations/.+}:cancel", rest.HandleCancelOperation).Methods("POST")
+	router.HandleFunc("/v1/{name:operations/.+}:wait", rest.HandleWaitOperation).Methods("GET")
```

Comment:
I took the approach of generating all the methods in the mixin protos. This means that if they are not referenced in the service config, their http bindings do not get overriden.

And adding `WaitOperation` to this service config causes a conflict with `EchoClient.WaitOperation`

Reply:
So we either live with the extra methods, or implement a strategy closer to that in the Go generator to ensure we don't generate the extra methods. Thoughts?